README.md

```diff
@@ -98,6 +96,6 @@ Making the following change reduces startup time exponentially, and gets rid of
 
 ## Authors
 
-Mike Bradley  
-Noi Sek  
-Tiffany Zhang  
+Mike Bradley
```

Comment:
Well, I'm glad you bring it up. I really think the line should look more like this:

```markdown
## Unified Borg Author

Bradley Sek Tiff Noi Mike Zhang
```

Thx and pls consider.